cypress/e2e/vendingMachine.cy.js

```diff
@@ -0,0 +1,248 @@
+
+          expect($tr.children()[PRODUCT_INDEX.NAME].textContent).to.equal(hot6.name);
+          expect(Number($tr.children()[PRODUCT_INDEX.PRICE].textContent)).to.equal(hot6.price);
+          expect(Number($tr.children()[PRODUCT_INDEX.QUANTITY].textContent)).to.equal(hot6.quantity);
+        });
+
+      const modifiedHot6 = { name: 'hot6', price: 1500, quantity: 5 };
+      cy.addProduct(modifiedHot6);
+
+      cy.get(SELECTOR.PRODUCT_INVENTORY_CONTAINER)
+        .children()
+        .each(($tr) => {
+          if ($tr.children()[PRODUCT_INDEX.NAME].textContent !== 'hot6') return;
+
+          expect($tr.children()[PRODUCT_INDEX.NAME].textContent).to.equal(modifiedHot6.name);
+          expect(Number($tr.children()[PRODUCT_INDEX.PRICE].textContent)).to.equal(modifiedHot6.price);
+          expect(Number($tr.children()[PRODUCT_INDEX.QUANTITY].textContent)).to.equal(modifiedHot6.quantity);
+        });
+    });
+
+    it('상품을 추가하고 나면 상품의 이름, 가격, 수량순으로 상품 목록이 보여진다.', () => {
+      const vitamin = { name: 'vitamin', price: 5000, quantity: 2 };
+      cy.addProduct(vitamin);
+
+      const hot6 = { name: 'hot6', price: 2000, quantity: 3 };
+      cy.addProduct(hot6);
+
+      const products = [vitamin, hot6];
+
+      cy.get(SELECTOR.PRODUCT_INVENTORY_CONTAINER)
+        .children()
+        .each(($tr, idx) => {
+          expect($tr.children()[PRODUCT_INDEX.NAME].textContent).to.equal(products[idx].name);
+          expect(Number($tr.children()[PRODUCT_INDEX.PRICE].textContent)).to.equal(products[idx].price);
+          expect(Number($tr.children()[PRODUCT_INDEX.QUANTITY].textContent)).to.equal(products[idx].quantity);
+        });
+    });
+
+    it('상품 목록은 탭을 이동하여도 기존의 상품 목록이 유지된다.', () => {
+      const vitamin = { name: 'vitamin', price: 5000, quantity: 2 };
+      cy.addProduct(vitamin);
+
+      const hot6 = { name: 'hot6', price: 2000, quantity: 3 };
+      cy.addProduct(hot6);
+
+      const products = [vitamin, hot6];
+
+      cy.get(SELECTOR.PRODUCT_INVENTORY_CONTAINER)
+        .children()
+        .each(($el, idx) => {
+          expect($el.children()[PRODUCT_INDEX.NAME].textContent).to.equal(products[idx].name);
+          expect(Number($el.children()[PRODUCT_INDEX.PRICE].textContent)).to.equal(products[idx].price);
+          expect(Number($el.children()[PRODUCT_INDEX.QUANTITY].textContent)).to.equal(products[idx].quantity);
+        });
+
+      cy.get(SELECTOR.VENDING_MACHINE_MANAGE_MENU).click();
+      cy.get(SELECTOR.PRODUCT_MANAGE_MENU).click();
+
+      cy.get(SELECTOR.PRODUCT_INVENTORY_CONTAINER)
+        .children()
+        .each(($el, idx) => {
+          expect($el.children()[PRODUCT_INDEX.NAME].textContent).to.equal(products[idx].name);
+          expect(Number($el.children()[PRODUCT_INDEX.PRICE].textContent)).to.equal(products[idx].price);
+          expect(Number($el.children()[PRODUCT_INDEX.QUANTITY].textContent)).to.equal(products[idx].quantity);
```

Comment:
```suggestion
          const { name, price, quantity } = products[idx];
          expect($el.children()[PRODUCT_INDEX.NAME].textContent).to.equal(name);
          expect(Number($el.children()[PRODUCT_INDEX.PRICE].textContent)).to.equal(price);
          expect(Number($el.children()[PRODUCT_INDEX.QUANTITY].textContent)).to.equal(quantity);
```
(마이너) 줄일 수 있는건 줄여봅시당 ㅎ.ㅎ

Reply:
저도 이러한 경우에 구조분해할당으로 객체에서 필요한 값들만 사용하는 방식을 좋아하는데 여기선 놓쳐버린 것 같습니다.
평소에도 객체에 접근하는 경우에는 스스로 한번 확인하거나 린트에서 잡아줬었는데 방심해버렸네요.
확인해주셔서 감사합니다!